### PR TITLE
always use fcUV2 in shapella even for non-proposer fcUs

### PR DESCRIPTION
**Pull request by tersec**

### CODE REVIEW

File: beacon_chain/beacon_node_light_client.nim [hunk 1]
```diff
@@ -60,15 +60,32 @@ proc initLightClient*(
...
-                payloadAttributes = NoPayloadAttributes)
+
+              template callForkchoiceUpdated(attributes: untyped) =
+                discard await node.elManager.forkchoiceUpdated(
+                  headBlockHash = blckPayload.block_hash,
+                  safeBlockHash = beaconHead.safeExecutionPayloadHash,
+                  finalizedBlockHash = beaconHead.finalizedExecutionPayloadHash,
+                  payloadAttributes = none attributes)
+
+              case node.dag.cfg.consensusForkAtEpoch(beaconHead.blck.bid.slot.epoch)
```
Comment (etan-status) [inline beacon_chain/beacon_node_light_client.nim:76]: `beaconHead.blck` is `nil` so this fails

Comment (etan-status) [reply beacon_chain/beacon_node_light_client.nim:76]: Should this be `blck.message.slot`, or simply based on wallSlot? @tersec

Comment (tersec) [reply beacon_chain/beacon_node_light_client.nim:76]: It's based on the timestamp of the referenced block

Comment (tersec) [reply beacon_chain/beacon_node_light_client.nim:76]: https://github.com/status-im/nimbus-eth2/pull/4835

**Pull request merged**